cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -741,10 +744,10 @@ def _create_patroni_config(self, patroni_config_path):
                 'password': pgsrv['cluster']['etcd']['patroni_password']
             },
         }
-        for node in pgsrv['cluster']['nodes']:
+        for node in pgsrv['cluster']['nodes'].values():
```

Comment:
maybe pgsrv -> db_server/postgres_server

Reply:
I agree it is not a good name, I assume it was chosen so lines like `676` and more will be one liner.

Reply:
I agree it is not a good name, I assume it was chosen so lines like 676 and more will be one liner.